tests/unit/models/reddit/test_wikipage.py

```diff
@@ -1,5 +1,7 @@
 import pickle
 
+import pytest
```

Comment:
Is this import necessary here?

Reply:
This has been removed in the latest commit.

Reply:
It was actually a remnant of a test that I had to later delete